src/doctokenizer.l

```diff
@@ -344,7 +344,7 @@ PHPTYPE   [\\:a-z_A-Z0-9\x80-\xFF\-]+
 CITESCHAR [a-z_A-Z0-9\x80-\xFF\-\?]
 CITEECHAR [a-z_A-Z0-9\x80-\xFF\-\+:\/\?]
 CITEID    {CITESCHAR}{CITEECHAR}*("."{CITESCHAR}{CITEECHAR}*)*|"\""{CITESCHAR}{CITEECHAR}*("."{CITESCHAR}{CITEECHAR}*)*"\""
-MAILADR   ("mailto:")?[a-z_A-Z0-9.+-]+"@"[a-z_A-Z0-9-]+("."[a-z_A-Z0-9\-]+)+[a-z_A-Z0-9\-]+
+MAILTOADDR   ("mailto:")?[a-z_A-Z0-9.+-]+"@"[a-z_A-Z0-9-]+("."[a-z_A-Z0-9\-]+)+[a-z_A-Z0-9\-]+
```

Comment:
There's a `MAILADDR` in another file in this directory. As I'm not sure if the files are related, I chose to give this a different name, but I think it's reasonable to argue that `adr` is a poor abbreviation, esp. given that `addr` is used for a similar token.

Reply:
Changing a "variable" name, in this case it is a symbolic pattern is dangerous, but looks like OK as it has been corrected on other places as well (problem probably originates from the fact that the English address is in Dutch adres).